modules/cloudsql-instance/variables.tf

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
This is empty line is what is causing the linter to fail. Revert the file and you'll be fine

Reply:
oh
done